Profile editor for the meshRoad object

lukaspj opened the pull request:
credit to Ryan Mounts (RDM)

found originally at http://www.garagegames.com/community/forums/viewthread/105391

Original PR at GarageGames repo:
https://github.com/GarageGames/Torque3D/pull/1105

Note from @Azaezel :
> As a note, the caps on the end of roads are not presently rendering with this one.

Azaezel commented:
want to say the main reason this one didn't get rolled in at the time was keybind shenanigans

lukaspj commented:
You also mentioned something about end of roads not rendering?
I dunno if I can figure out how to test this, but I will try

lukaspj committed:
Profile editor for the meshRoad object

credit to Ryan Mounts (RDM)

found originally at http://www.garagegames.com/community/forums/viewthread/105391

lukaspj commented:
Seems to work as expected for me, the controls are a bit wonky, but otherwise seems to function as expeceted
